### ISSUE TITLE
CCDScan: auto refresh in CCDScan Account's sidebar info. doesn't happen, user has to reload CCDScan again in order to view changes

### ISSUE THREAD

**Issue by SRConcordium** [Concordium/concordium-scan#22]
CCDScan: auto refresh in CCDScan Account's sidebar info. doesn't happen, user has to reload CCDScan again in order to view changes

https://stagenet.ccdscan.io/

Change in Account sidebar related info. e.g. new transactions etc. doesn't reflect unless CCDScan page is reloaded again and then account page is reopened. This doesn't look to be a good user experience. New info. should refresh automatically without having to reload CCDScan site.

**Comment by parv0888** [comment]
Duplicate of 
https://github.com/Concordium/concordium-scan/issues/31

**Comment by mh-concordium** [comment]
Tested in  version 0.1.0/1.0.1. I was not able to confirm the change, issue seems to be persistent. I had to refresh the page to see the changes.
https://images.zenhubusercontent.com/495930068/51960e76-8f36-49f8-abab-8a2ac5983f9f/refreshbeforechange.mp4

**Comment by SRConcordium** [comment]
This is still an issue on Stagenet ([version 0.1.2/1.0.2](https://stagenet.ccdscan.io/)), https://stagenet.ccdscan.io/?dcount=1&dentity=account&daddress=38TyuhoFn4SY4JyWjPJGLriyn5DBQk7QdnFLcRBB6zvi983gyp
Steps to replicate the issue - 
Open Account statement sidebar for an account in CCDScan - open wallet . make a new transaction - let the transaction complete - close and reopen Account Statement sidebar in CCDScan without reloading the account - (Actual behavior) - The new transaction still doesn't show up in Account Statement. It starts showing only after CCDScan page is refreshed.
